src/sentry/static/sentry/app/components/group/sidebar.jsx

```diff
@@ -202,8 +216,9 @@ const GroupSidebar = createReactClass({
     return (
       <div className="group-stats">
         <SuggestedOwners event={this.props.event} />
-
-        <GroupReleaseStats group={group} location={this.context.location} />
+        {this.state.allEnvironmentsGroupData && (
```

Comment:
maybe consider rendering the component with empty data here as a loading state? this fix is good either way so maybe it could be left as a possible followup?

Reply:
👍 This component needs quite a bit of work in general, but will ship this first as it's in a broken state currently (if you have the feature on)